### PR TITLE
Feature/internationalization

### PR DESCRIPTION
**Pull request by lionellbriones**
Notes
- Run import locales > node scripts/importLocales.js
- Locales json files are located > app/src/plugins/i18n

### CODE REVIEW

File: app/scripts/importLocales.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+const https = require('https')
+var fs = require('fs')
+
+const apiKey = process.env.AIRTABLE_API_KEY || 'keyZ491mmTxAHPvMC'
```
Comment (chaitanyapotti) [inline app/scripts/importLocales.js]: can't put this on the frontend

**Comment by chaitanyapotti** [comment]
what is the status of this.?
Please address the issues. Let's release this asap in v0.2.12

File: app/scripts/importLocales.js (hunk 1 above)
Comment (lionellbriones) [reply app/scripts/importLocales.js]: Done.. moved the api to backend

File: app/src/plugins/vuetify.js [hunk 2]
```diff
@@ -6,6 +6,12 @@ import { THEME_DARK_BLACK_NAME, THEME_LIGHT_BLUE_NAME } from '../utils/enums'
 
 Vue.use(Vuetify)
 
+import { languages } from './i18n'
+
+let userLanguage = window.navigator.userLanguage || window.navigator.language
+userLanguage = userLanguage.split('-')
+userLanguage = userLanguage[0]
+
```
Comment (chaitanyapotti) [inline app/src/plugins/vuetify.js]: check if we support this language. if not, fallback to en

Comment (lionellbriones) [reply app/src/plugins/vuetify.js]: > check if we support this language. if not, fallback to en

Done

**Pull request merged**